### ISSUE TITLE
No add blocking in roaming

### ISSUE THREAD

**Issue by Konfetti030** [AdguardTeam/AdguardForAndroid#4749]
### Please answer the following questions for yourself before submitting an issue.

- [X] I am running the latest version
- [X] I checked the documentation and found no answer
- [X] I checked to make sure that this issue has not already been filed

### AdGuard version

4.0 nightly 110

### Environment

```markdown
- OS: A13
- Device: Xiaomi Mi 11 Pro
- Firmware: Xiaomi.eu MIUI 14 ROM
```

### Root access

- [ ] Yes, I have it.

### What filters do you have enabled?

AdGuard Base filter, AdGuard Mobile Ads filter, AdGuard German filter, AdGuard Tracking Protection filter, AdGuard URL Tracking filter, AdGuard Social Media filter, AdGuard Annoyances filter, AdGuard Cookie Notices filter, AdGuard Popups filter, AdGuard Mobile App Banners filter, AdGuard Widgets filter, AdGuard Other Annoyances filter, AdGuard DNS filter, EasyList Germany

### What Stealth Mode options do you have enabled?

Send Do-Not-Track signals

### Issue Details

Steps to reproduce:
1. Connection to 4G WiFi router or in roaming internet connection
2. Surfing domestic websites like at home (German language)
3. Ads don't get blocked but same filters (language specific etc.) are activated in roaming like at home with domestic connection.

### Expected Behavior

Ads are getting blocked in roaming like at home.

### Actual Behavior

Surfing same websites like at home but no ads get blocked nor in WiFi or roaming internet connection.
On main page ads get blocked but not in sub websites.

### Screenshots

<details><summary>Screenshot 1:</summary>

</details>

### Additional Information

Main page:
https://m.spox.com/de/index.html

Example (at the bottom all ads are shown):
https://m.spox.com/de/sport/listen/fussball/bundesliga/fc-bayern/2305/fc-bayern-vor-naechstem-neustart-finden-uli-hoeness-karl-heinz-rummenigge-endlich-genehme-nachfolger-ftr.html

**Comment by Versty** [comment]
@Konfetti030 Hi! 
Could you provide the logs from your device? 

1.  Enable debug logging: 
*Settings -> General -> Advanced -> Logging level -> Debug*

2. Reproduce the issue, then remember the exact time when it happened 

3. Collect logs: 
*Settings -> General -> Advanced -> Export logs and system info* 

4. Sent this file to devteam@adguard.com:
      - mention the `4749` number in the subject
      - specify the exact time when the issue occurred

**Comment by Konfetti030** [comment]
Sent you debug log a few days ago

**Comment by Versty** [comment]
@Konfetti030 Hi! 
We have made some improvements since your report. Does the issue persist on the latest release version AdGuard For Android?

**Comment by Konfetti030** [comment]
@Versty I will let you know in a few weeks when I will be in roaming again. But I think problem will be solved due to #153460 outbrain filters.

**Comment by Versty** [comment]
@Konfetti030 If the problem still persists the next time you test, please reopen this issue.
